.vuepress/styles/index.styl

```diff
@@ -6,3 +6,13 @@
 
 .home .hero img
   max-width 450px!important
+
+.guilabel
+  border 1px solid #7fbbe3
+  background #e7f2fa
+  font-size 80%
+  font-weight 700
+  border-radius 4px
+  padding 2.4px 6px
+  margin auto 2px
+  white-space nowrap
```

Comment:
Not convinced we need this extra style. For literal buttons, things between backticks seem good enough?

Reply:
Removing the style and using backticks sometimes causes wrapping as seen below:
![image](https://user-images.githubusercontent.com/48768719/91548675-07f8db00-e926-11ea-8102-dedc6ac6fde1.png)

Reply:
Is that a problem? You can always use non-breaking spaces if it needs to be on a single line

Reply:
Ooh cool, I didn't know that was a thing.